sklearn/neural_network/multilayer_perceptron.py

```diff
@@ -828,6 +832,15 @@ class MLPClassifier(BaseMultilayerPerceptron, ClassifierMixin):
 
         .. versionadded:: 0.20
 
+    max_fun : int, optional, default 15000
+        Only used when solver='lbfgs'. Maximum number of function calls.
+        The solver iterates until convergence (determined by 'tol'), number
+        of iterations reaches max_iter, or this number of function calls.
```

Comment:
I think it makes sense here to say which "function" call. this is a bit confusing right now.

Reply:
This function is just the loss function of the NN that is being optimized.  I've clarified this in the comments by chaning "function" to "loss function".  Please advise if this addresses the confusion.

Reply:
@adrinjalali please advise if the clarified documentation comments address your concern.  If so please resolve this conversation and approve the commit.